packages/v4/package.json

```diff
@@ -21,7 +21,8 @@
     "@patternfly/quickstarts": "^2.2.4",
     "@patternfly/react-docs": "5.96.8",
     "react": "^16.8.0 || ^17.0.0",
-    "react-dom": "^16.8.0 || ^17.0.0"
+    "react-dom": "^16.8.0 || ^17.0.0",
+    "@ausuliv/react-log-viewer": "^4.84.2"
```

Comment:
what changes are in your forked version? Shouldn't bring in forked versions if possible

Reply:
No changes, the forked version is the extracted log viewer pulled out of the pf-react repo into its own repo as that was the main goal here.